packages/flutter/test/widgets/listener_test.dart

```diff
@@ -109,6 +112,7 @@ void main() {
           onPointerExit: (PointerExitEvent details) => exit = details,
         ),
       ));
+      final RenderPointerListener renderListener = listenerKey.currentContext.findRenderObject();
```

Comment:
Here, you can use a finder:
```suggestion
      final RenderPointerListener renderListener = tester.renderObject(find.byType(Listener));
```
That way you don't need the GlobalKey.